rosdep/python.yaml

```diff
@@ -1648,6 +1648,13 @@ python-fcn-pip: &migrate_eol_2025_04_30_python3_fcn_pip
     pip:
       depends: [liblapack-dev]
       packages: [fcn]
+python-filetype-pip:
```

Comment:
there only seems to be a python3 version of the package (https://pypi.org/project/filetype/), but this is a python 2 entry.
If you wanted to add this for python3, please use `python3-filetype-pip`,

Reply:
Correct. Resolved

Reply:
@bullerbeck it should be `python3-filetype-pip`.